app/routers/event.py

```diff
@@ -81,20 +87,26 @@ async def create_event_api(event: EventModel, session=Depends(get_db)):
     return {"success": True}
 
 
+def get_categories_list(db_session: Session = Depends(get_db)):
```

Comment:
Where it was before? 
I think it should be in `internal/`

Reply:
Actually, I didn't write this function nor made any use of it... but Yam asked me to add the user integration so I did:)
I will push it soon:)